src/balance_queue.py

```diff
@@ -125,7 +124,7 @@ def sell(self, change: decimal.Decimal) -> Optional[list[transaction.SoldCoin]]:
                 return None
 
             not_sold = bop.op.change - bop.sold
-            assert not_sold > 0
+            assert not_sold >= 0
```

Comment:
bop should have been removed from the queue, if it has no coins left which can be sold. So this should be > 0. Otherwise we made a mistake earlier.

Reply:
I reverted the commit which added this change. Will try to find some time and check why this happens on my side then. 👍